.circleci/test-deploy.yml

```diff
@@ -158,7 +174,9 @@ workflows:
             - Standard Npm Example
             - Custom Node Version Example
             - Yarn Example
+            - Yarn Version Example
             - Pnpm Example
+            - Pnpm Version Example
```

Comment:
`Pnpm Version Example` is not defined 

Are you using the [VSCode CircleCI extension](https://circleci.com/docs/vs-code-extension-overview/) to catch errors in your syntax?

Reply:
I use jetbrains, so unfortunately no circleci plugin or extension